BlueShift-iOS-SDK/BlueShiftAppDelegate.m

```diff
@@ -682,7 +691,7 @@ - (void)handleCarouselPushForCategory:(NSString *)categoryName usingPushDetailsD
                 }
             }
             
-            [self setupPushNotificationDeeplink: selectedItem];
+            [self setupPushNotificationDeeplink: pushDetails];
```

Comment:
`selectedItem` will not be used anymore?

Reply:
Yes, this is not used anywhere. The selectedItem is the hashmap of clicked carousel image from the carousel array. In order to share the full payload, I have replaced it with pushDetails.

Reply:
by any chance, this will be used by the host application? if they override the push clicks

Reply:
SelectedItem was only used for deep link processing. So this will not be used by host application.

Reply:
if this does not affect the host app, I'm good with this change.